app/models/user_access.rb

```diff
@@ -93,7 +93,7 @@ def accessible_admins(action)
   end
 
   def accessible_users(action)
-    return User.none unless [:manage, :view_reports].include?(action)
+    return User.none unless [:manage, :view_reports, :view_pii].include?(action)
```

Comment:
Nice. Is this an unrelated fix?

Reply:
Not a 'fix' per se, this adds the `view_pii` action for `accessible_users`. Needed it for the `users#show` view.